android/app/src/main/java/app/organicmaps/util/Utils.java

```diff
@@ -226,6 +226,19 @@ public static void openAppInMarket(Activity activity, String url)
     }
   }
 
+  /*
+   * Check if WebBrowser intent could be opened.
+   */
+  public static boolean isBrowserAvailable(Context context)
+  {
+    final Intent intent = new Intent(Intent.ACTION_VIEW);
+    intent.setData(Uri.parse("https://osm.org"));
```

Comment:
Could it be that `Uri.parse(OsmOAuth.nativeGetOAuth2Url()` behaves differently compared to `Uri.parse("https://osm.org")`? @strump can you please modify this function to specifically check for OSM auth, so we can test it with a user?